### PR TITLE
DYN-5742 add workspace

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose

Graph is not being added in the workspace because dyn file was not found.
The problem is that when executing DynamoSandbox from a path that contain empty spaces the URI.AbsolutePath is converting them to "%20" so the dyn file is not found.
Then with this fix I'm removing the "%20" from the string path and is found correctly when inserting the graph in the workspace.

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [X] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

Graph is not being added in the workspace because dyn file was not found.

### Reviewers

@QilongTang 

### FYIs

@reddyashish

### CODE REVIEW

File: src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs [hunk 1]
```diff
@@ -448,8 +445,7 @@ internal void InsertGraph()
         private string DynamoGraphFromMDFilePath(string path)
         {
             var dynPath = Path.Combine(Path.GetDirectoryName(path), Path.GetFileNameWithoutExtension(path).Replace("%20", " ")) + ".dyn";
-            return dynPath;
-            return path.Replace("%20", " ").Replace(".md", ".dyn");
+            return dynPath.Replace("%20", " ");
```
Comment (mjkkirschner) [inline src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs:448]: I think it is more robust to use a url decode method https://learn.microsoft.com/en-us/dotnet/api/system.web.httputility.urldecode?source=recommendations&view=net-7.0

**Comment by RobertGlobant20** [comment]
GIF showing that the graph is added to workspace when Dynamo is executed from a path that contains empty spaces.
AddingGraphToWorkspace_EmptySpaces

File: src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs (hunk 1 above)
Comment (RobertGlobant20) [reply src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs:448]: @mjkkirschner even if the DynamoGraphFromMDFilePath method is called from 2 different places I will need to use the UrlDecode method is both places, is that ok?

Comment (RobertGlobant20) [reply src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs]: Fixed in the next commit: https://github.com/DynamoDS/Dynamo/pull/13863/commits/5c3666153466e7fa04b950918fd7f7b46664a79f

**Review by reddyashish — APPROVED** [review]

**Comment by reddyashish** [comment]
@RobertGlobant20 Can you add a test for this.

**Comment by QilongTang** [comment]
From our internal discussion https://autodesk.slack.com/archives/C1866AFQB/p1680717374613279, merging this and continuing to work once @RobertGlobant20 is back.

**Pull request merged**